chrome/content/zotero/zoteroPane.xul

```diff
@@ -106,7 +106,19 @@
 					<toolbarbutton id="zotero-tb-library-add-menu" class="zotero-tb-button" tooltiptext="&zotero.toolbar.newLibrary.label;" type="menu">
 						<menupopup id="zotero-tb-library-add-popup">
 							<menuitem id="zotero-tb-group-add" label="&zotero.toolbar.newGroup;" oncommand="ZoteroPane_Local.newGroup()"/>
-							<menuitem id="zotero-tb-feed-add" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
+							<menu id="zotero-tb-feed-add-menu" label="&zotero.toolbar.feeds.new;">
```

Comment:
Ah. Well I'm working on a feed-discovery branch which adds another menuitem there and kind of have it at the back of my head, so I didn't even think about it this way. I could hide it for this branch, but probably not much point?